tests/.sauce/config_on_sauce.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1alpha
```

Comment:
Would recommend just calling this file `config-win.yml`, since there is no other way to run this than sauce (no docker).

Reply:
ok 👍